### DISCUSSION TITLE
Add the option "Go To Top" to the "Floating Action Button"

### DISCUSSION THREAD

**Discussion by huubert** [Docile-Alligator/Infinity-For-Reddit#1223] [Completed Ideas] [answered]
It would be quite useful to just easily go to the top of a subreddit after you already scrolled down a lot.

**Comment by Docile-Alligator** [comment, marked as answer]
Done https://github.com/Docile-Alligator/Infinity-For-Reddit/commit/b74fc9354c12df9ca03448a04df21c1fcc2b8d4e
